### DISCUSSION TITLE
Does Mirth Connect Supports Conversion of C-CDA to FHIR?

### DISCUSSION THREAD

**Discussion by MaheshBalasubramanian** [nextgenhealthcare/connect#5920] [Q&A]
Is it Possible to convert C-CDA to FHIR in Mirth Connect? If Yes, can conversion be achieved in Mirth Connect without adding external libraries.

**Comment by pacmano1** [comment]
Yes, but there is no easy button.  You have to build the transformation yourself.

Of course, your question does not exactly define what you mean by convert to FHIR.
